ts/features/idpay/initiative/details/components/TimelineDetailsBottomSheet.tsx

```diff
@@ -230,13 +136,29 @@ export const useTimelineDetailsBottomSheet = (
 
   const modal = useIOBottomSheetModal(
     <TimelineDetailsBottomSheet />,
-    I18n.t("idpay.initiative.operationDetails.title"),
-    530,
+    modalConfig.title,
```

Comment:
```js
  const calculateHeight = (e: LayoutChangeEvent) => {
    const { height } = e.nativeEvent.layout;
    setModalConfig({ ...modalConfig, snapPoint: 150 + height });
  };

  const modal = useIOBottomSheetModal(
    <View style={IOStyles.flex} onLayout={e => calculateHeight(e)}>
      <TimelineDetailsBottomSheet />
    </View>
```
I believe opting for a more generic approach to bottom sheet sizing would provide more scalability and less imperative code, this also goes hand in hand with a refactor of the snapPoint/Title map to only include an I18n key

Reply:
With this solution the modal will change its height multiple times during rendering, resulting in a "weird" bounce. Better have fixed heights here!